Source/Blazorise/Components/FileEdit/FileEdit.razor.cs

```diff
@@ -253,6 +253,15 @@ public ValueTask RemoveFile( int fileId )
             return JSFileEditModule.RemoveFile( ElementRef, ElementId, fileId );
         }
 
+        /// <summary>
+        /// Show a browser picker for file input.
+        /// </summary>
+        /// <returns>A task that represents the asynchronous operation.</returns>
+        public Task ShowPicker()
+        {
+            return JSUtilitiesModule.ShowPicker( ElementRef, ElementId ).AsTask();
```

Comment:
Isn't this the same as the regular existing `Open` for the `FileEdit` ?

Reply:
We don't have `Open` on FileEdit.

Reply:
It is not directly exposed. It's in the module. We had a user complaining he'd access the module and couldn't call `Open` before. With this, I guess we have two ways to open it?
https://github.com/Megabit/Blazorise/pull/4204

Reply:
Oh, that. Then the JSmodule version doesn't make much sense anymore.